engine/node_matlab.cc

```diff
@@ -493,6 +493,41 @@ namespace MathDAG
     checkArg(0,0);
     return o<<"frac("<<arguments[0][0]->matlab()<<")";
   }
+  
+  template <>
+  ostream& OperationDAG<OperationType::percent>::matlab(ostream& o) const
+  {
+    checkArg(0,0);
+    return o<<"100*("<<arguments[0][0]->matlab()<<")";
+  }
+ 
+  template <>
+  ostream& OperationDAG<OperationType::gamma>::matlab(ostream& o) const
+  {
+    checkArg(0,0);
+    return o<<"gamma("<<arguments[0][0]->matlab()<<")";
+  }
+  
+  template <>
+  ostream& OperationDAG<OperationType::digamma>::matlab(ostream& o) const
+  {
+    checkArg(0,0);
+    return o<<"psi(2,("<<arguments[0][0]->matlab()<<"))";
```

Comment:
The digamma function is psi(0,x)

Reply:
In matlab, you can omit the order if it is the digamma function, ie psi(x) == psi(0,x)

Reply:
I've removed digamma because it is no longer needed if I already have polygamma(0,x).

Reply:
Sounds good.